### ISSUE TITLE
miner finds only 4 form 9 P106-100 cards

### ISSUE THREAD

**Issue by mannebk79** [Lolliedieb/lolMiner-releases#1882]
Im on a freshly installed debian bullseye with lol 1.69, got the nvidia driver-packet from contrib/non-free installed, nvidia-detect finds all cards, lolminer finds only first 4 cards. 

Any ideas whats wrong? 
Any ideas how to fix or troubleshoot?

Thanks
Manne

**Comment by jgonzis** [comment]
Could you show some extra information with photos

**Comment by mannebk79** [comment]
sure thing

lol runing with 4 units:

![grafik](https://user-images.githubusercontent.com/60316272/223733207-09ca1e96-6330-40f9-89fd-606756907644.png)

and output of nvidia-detect

![grafik](https://user-images.githubusercontent.com/60316272/223733390-5f80ad38-7fd6-4bba-9a16-b810c9b713f7.png)

and nvidia-smi

![grafik](https://user-images.githubusercontent.com/60316272/223734347-c109f15c-c92f-4aed-9e46-bea8ba7eccdc.png)

**Comment by jgonzis** [comment]
Did you enable 4G in Bios

**Comment by mannebk79** [comment]
yes of course, as i said, it did work with other linux based os. btw what disto do you recomend with your lolminer?

![photo_2023-03-08_17-07-54](https://user-images.githubusercontent.com/60316272/223766426-14d16e8a-be54-4e5f-b61d-8ef011911ca1.jpg)
